src/main.py

```diff
@@ -93,23 +120,24 @@ def process_file(filename):
     else:
         raise(f"BEAT_TO_NOTE_VERSION '{BEAT_TO_NOTE_VERSION}'' not recognized")
 
-    logger.debug(f"Meter: {meter}")
-
-    logger.debug("Sounds:")
-    for sound in sounds:
-        logger.debug(f"{sound.timestamp:.3f}: {sound.symbol}\t{sound.duration_ms:.3f} ({sound.rhythmic_value})")  # noqa
-
-    tonation = tonation_recognition.get_tonation(sounds)
-
-    chords = chords_generation.get_chords(sounds, tonation, (4, 8))
+    if args.tonation:
+        if args.tonation.islower():
+            kind = "minor"
+        else:
+            kind = "major"
+        tonation = music.Tonation(symbol=args.tonation.lower(), kind=kind)
+    else:
+        tonation = tonation_recognition.get_tonation(sounds)
 
-    logger.debug("Chords:")
-    for chord in chords:
-        logger.debug(f"{str(chord).ljust(20)}\t{chord.duration:.3f}")  # noqa
+    chords = chords_generation.get_chords_daria(sounds, tonation, (4, 4))
 
     result_file = music_synthesis.create_midi("output.midi", sounds, chords)
-    
-    logger.debug(f"Result file: {result_file}")
+    result_file = music_synthesis.save_midifile_as_wav("output.midi", "output.wav")
```

Comment:
This does not work on both of my computers. Comment it out if you want to merge this
![image](https://user-images.githubusercontent.com/47048420/99911893-a1524600-2cf7-11eb-9327-2ec796cbeea0.png)